docs/development-setup.md

```diff
@@ -17,8 +17,8 @@ We'll use the `oc cluster up` method, since it's very easy to use:
     ```
     $ oc cluster up
     ```
-    (If you are running into some connection issues, please flush your iptables
-    rules bufore starting the cluster with `iptables -F`)
```

Comment:
I did some more testing and with the firewall rules added, things seemed to be robust when I did 'oc cluster down', 'oc cluster up', changed IP, rebooted, etc. No firewall problems. I'm hestitant to recommend iptables -F because I was having a painful time when I was trying to use it at the recommendation of this document - one thing would start working, something else would break, I'd need to restart docker, etc. I think there are rules in the FORWARD chain that are actually needed for correct operation. But I've added some wording suggesting iptables -F as a last resort. See what you think.